### DISCUSSION TITLE
How to configure RFID Reader

### DISCUSSION THREAD

**Discussion by jmt-cash** [MiczFlor/RPi-Jukebox-RFID#1829] [Q&A]
Hello,

I know that my problem is not directly a Phoniebox problem, but I hope that even though someone can help me. My RFID Reader suddenly startet to give out values like: ö4d22980a_e when I scan one of the blue key dongles. I am aware that the device has different output types, but I dont know how to change them. In an amazon add I found the instructions to disconnect and reconnect the device twice so that it switches zu Konfiguration mode, but that does not work...
Has anybody an idea on how to reach management mode or how to configure the device directly from the Pi?

Yours

**Comment by Andreas-Roth-27** [comment]
Where is that instruction to disconnect and reconnect the reader, please? I am asking because the reader that I have (taken from the shopping list) does not work with the phoniebox. It makes the RPi beep and display the id of the blue chip. But it does not have any influence on the phoniebox
